commands/priority.go

```diff
@@ -9,8 +9,9 @@ import (
 )
 
 const (
-	openFaaSURLEnvironment = "OPENFAAS_URL"
-	templateURLEnvironment = "OPENFAAS_TEMPLATE_URL"
+	openFaaSURLEnvironment      = "OPENFAAS_URL"
+	templateURLEnvironment      = "OPENFAAS_TEMPLATE_URL"
+	templateStoreURLEnvironment = "OPENFAAS_TEMPLATE_STORE_URL"
```

Comment:
Please can you document each of these three env-vars in the docs and in this repo's README file? Happy for you to open a new issue to cover that work.

Reply:
Will probably open a new Issue for this

Reply:
Issue as pointed ^

Reply:
Actually all three of those are documented in `README.md`

Reply:
I will open an issue for `openfaas/docs`

Reply:
ok added documentation in `TEMPLATES.md` on the new feature also